src/values.rs

```diff
@@ -443,9 +444,9 @@ impl From<Range<f32>> for JitteredValue {
 ///
 /// ```
 /// # use bevy_particle_systems::values::Lerpable;
-/// # use bevy::prelude::Color;
+/// # use bevy::prelude::{Color};
 /// assert_eq!(0.0_f32.lerp(1.0, 0.5), 0.5);
-/// assert_eq!(Color::WHITE.lerp(Color::BLACK, 0.5), Color::rgba(0.5, 0.5, 0.5, 1.0));
+/// assert_eq!(Color::WHITE.lerp(Color::BLACK, 0.5), Color::linear_rgba(0.5, 0.5, 0.5, 1.0).into());
```

Comment:
Shouldn't this be `Color::srgba()` instead?

Reply:
I don't know why, but the Bevy team decided to use linear space for `Color::WHITE/BLACK` so the test will fail with srgb. (The values *will* be pretty close, but due to conversion errors there's a slight difference)